EOLib/PacketHandlers/Walk/WalkPlayerHandler.cs

```diff
@@ -11,10 +11,10 @@
 
```

Comment:
Looks like nothing actually changed here, please reset to the content of the `master` branch